storybook/stories/chart/areachart/AreaChart.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { format, parseISO, subDays } from "date-fns";
+import { ComponentMeta } from '@storybook/react';
+import { Area, AreaChart, CartesianGrid, ResponsiveContainer, Tooltip, XAxis, YAxis } from '../../../../src';
+import { TimeSeriesData } from '../../data';
+
+export default {
+  title: 'Recharts/Chart/Area Chart',
+  component: AreaChart,
+} as ComponentMeta<typeof AreaChart>;
+
+const data: TimeSeriesData[] = [];
+for (let num = 30; num >= 0; num--) {
+  data.push({
+    date: subDays(new Date(), num).toISOString().substring(0, 10),
+    value: 1 + Math.random(),
+  });
+}
+
+export const Simple = {
+  render: (args: any) => {
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <AreaChart data={data}>
+          <defs>
+            <linearGradient id="color" x1="0" y1="0" x2="0" y2="1">
+              <stop offset="0%" stopColor="#2451B7" stopOpacity={0.4} />
+              <stop offset="75%" stopColor="#2451B7" stopOpacity={0.05} />
+            </linearGradient>
+          </defs>
```

Comment:
Could you please split this one story up into separate ones, each showing some feature? 

(1) Most simple
(2) with Gradient
(3) ... any other ideas you have.